runtime/v1/shim/service.go

```diff
@@ -547,23 +544,27 @@ func (s *Service) checkProcesses(e runc.Exit) {
 	}
 }
 
-func shouldKillAllOnExit(bundlePath string) (bool, error) {
+func shouldKillAllOnExit(bundlePath string, ctx context.Context) bool {
```

Comment:
basically, context should be first argument.

Reply:
Context is now first argument.